addons/mrp/views/mrp_bom_views.xml

```diff
@@ -90,7 +90,7 @@
                                     <field name="parent_product_tmpl_id" invisible="1" />
                                     <field name="possible_bom_product_template_attribute_value_ids" invisible="1"/>
                                     <field name="product_uom_id" options="{'no_open':True,'no_create':True}" groups="uom.group_uom"/>
-                                    <field name="bom_product_template_attribute_value_ids" widget="many2many_tags" options="{'no_create': True}" attrs="{'column_invisible': [('parent.product_id', '!=', False)]}" groups="product.group_product_variant"/>
+                                    <field name="bom_product_template_attribute_value_ids" optional="hidden" widget="many2many_tags" options="{'no_create': True}" attrs="{'column_invisible': [('parent.product_id', '!=', False)]}" groups="product.group_product_variant"/>
```

Comment:
hide and hidden work the same ? pfff misère

Reply:
Yep, but as `hide` is more used, I will use this one instead :+1: